scenes/inventory.py

```diff
@@ -16,7 +16,7 @@
 class InventoryScene(BaseScene):
```

Comment:
See above comment about how this probably should be a separate scene but a window in the GameScene.